src/openads-appnexus/service/DomainEventBus.js

```diff
@@ -0,0 +1,71 @@
+import {observerErrorThrown} from './observerErrorThrown'
+
+class DomainEventBus {
```

Comment:
I know you are still working on it but can I suggest a name change in this class?
IMO fits better here ReplayEventBus, there is no "domain" in this library...